packages/toolkit/src/index.ts

```diff
@@ -23,13 +23,14 @@ export const theme = {
   },
 };
 
-export type ISize = 'xs' | 's' | 'm' | 'l' | 'xl' | 'xxl';
+export type ISize = 'xs' | 's' | 'm' | 'l' | 'lt' | 'xl' | 'xxl' | 'xxxl';
```

Comment:
xxxl скорее всего нужен только для аватара, т.к. под него нет класса `.size-#{$size}` и переменных
```suggestion
export type ISize = 'xs' | 's' | 'm' | 'l' | 'lt' | 'xl' | 'xxl';
```